<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TvoSky</class>
 <widget class="QMainWindow" name="TvoSky">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Welcome Using Skype!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Skype Account</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string>drswinghead</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;a href=&quot;http://nullget.sf.net&quot;&gt;No Skype Account Yet?&lt;/a&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;a href=&quot;http://nullget.sf.net&quot;&gt;Forgot password?&lt;/a&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Auto login when start</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../icons/tvosky.qrc">
             <normaloff>:/icons/status/enteredtext.png</normaloff>:/icons/status/enteredtext.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>      Log in      </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>46</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../icons/tvosky.qrc">:/icons/network-disconnect.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="currentIndex">
            <number>8</number>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>ONLINE</string>
            </property>
            <property name="icon">
             <iconset resource="../icons/tvosky.qrc">
              <normaloff>:/icons/status/available.png</normaloff>:/icons/status/available.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SKYPEME</string>
            </property>
            <property name="icon">
             <iconset resource="../icons/tvosky.qrc">
              <normaloff>:/icons/status/skypeme.png</normaloff>:/icons/status/skypeme.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AWAY</string>
            </property>
            <property name="icon">
             <iconset resource="../icons/tvosky.qrc">
              <normaloff>:/icons/status/idle.png</normaloff>:/icons/status/idle.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BUSY</string>
            </property>
            <property name="icon">
             <iconset resource="../icons/tvosky.qrc">
              <normaloff>:/icons/status/unknown.png</normaloff>:/icons/status/unknown.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NOTCO</string>
            </property>
            <property name="icon">
             <iconset resource="../icons/tvosky.qrc">
              <normaloff>:/icons/status/away.png</normaloff>:/icons/status/away.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>INVISIBLE</string>
            </property>
            <property name="icon">
             <iconset resource="../icons/tvosky.qrc">
              <normaloff>:/icons/status/invisible.png</normaloff>:/icons/status/invisible.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OFFLINE</string>
            </property>
            <property name="icon">
             <iconset resource="../icons/tvosky.qrc">
              <normaloff>:/icons/status/offline.png</normaloff>:/icons/status/offline.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>--------------</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PNS</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QListWidget" name="listWidget_2">
          <property name="editTriggers">
           <set>QAbstractItemView::AllEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
         <widget class="QWidget" name="">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBox_2">
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_3"/>
              </item>
              <item row="1" column="2">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>89</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>435</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVvvvvv">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionVvvvv"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuVvvvvv"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVvvvv">
   <property name="text">
    <string>vvvvv</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/tvosky.qrc"/>
 </resources>
 <connections/>
</ui>
